src/engine/Scene.hpp

```diff
@@ -22,12 +22,26 @@ class Scene : public renderer::IDrawable {
 
     std::multiset<std::shared_ptr<GameObject>, GameObjectComparator> objects;
```

Comment:
I know Jakob added this, but perhaps we should discuss this in our next meeting.